### ISSUE TITLE
Human in the loop masks from _seg.npy file 

### ISSUE THREAD

**Issue by Dana162001** [MouseLand/cellpose#787]
Hi, I annotated some images with cellpose and then manually added a few masks using human in the loop interface. What I got is _seg.npy file that contains 'masks' but also information about 'manual_changes': such as 'added mask' and 'removed mask', my question is does 'masks' automatically updated and have information about all manual changes? Or I have to modify it to add all my manual changes before converting to .png file? (I need .png file to run ground truth quality control script). And if this information has to be added, are there any available scripts to do that? Thank you in advance for your answer!

**Comment by mrariden** [comment]
No, you don't need to edit the .npy file. When you export the .png file, it will be the same as the GUI viewer.
